articles/055_ros_parameter_design.md

```diff
@@ -109,17 +109,28 @@ Each parameter consists of a key and a value.
 The key is a string value.
 The value can be one of the following datatypes:
 
-- `float64`
```

Comment:
I rearranged these to match the order of specification in `rcl_interfaces` as in the `rclcpp` code.